instrumentation/finagle-http-23.11/javaagent/src/main/java/io/netty/channel/ChannelInitializerDelegate.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.netty.channel;
+
+/** Exists to correctly expose and propagate the {@link #initChannel(Channel)} calls. */
+public abstract class ChannelInitializerDelegate<T extends Channel> extends ChannelInitializer<T> {
```

Comment:
(just to namespace it a bit since it's not under our package)
```suggestion
public abstract class OpenTelemetryChannelInitializerDelegate<T extends Channel> extends ChannelInitializer<T> {
```